docs/reference/setup/install/docker.asciidoc

```diff
@@ -202,7 +203,17 @@ endif::[]
 To stop the cluster, type `docker-compose down`. Data volumes will persist, so it's possible to start the cluster again with the same data using `docker-compose up`.
 To destroy the cluster **and the data volumes** just type `docker-compose down -v`.
 
-===== Inspect status of cluster:
```

Comment:
Note that the Elastic docs are in the asciidoc format, so markdown isn't supported. An introduction to this format can be found in: https://github.com/elastic/docs